modules/infra/submodules/storage/netapp.tf

```diff
@@ -154,7 +154,7 @@ resource "aws_fsx_ontap_storage_virtual_machine" "eks" {
 resource "aws_fsx_ontap_volume" "eks" {
   count                      = local.deploy_netapp && var.storage.netapp.volume.create ? 1 : 0
   storage_virtual_machine_id = aws_fsx_ontap_storage_virtual_machine.eks[0].id
-  name                       = "${var.deploy_id}_${var.storage.netapp.volume.name_suffix}"
+  name                       = replace("${var.deploy_id}_${var.storage.netapp.volume.name_suffix}", "-", "_")
```

Comment:
Should we do a regex substitute instead replacing `[^a-z_]`?

Reply:
sure

Reply:
Sorry, I should have given a full alphanumeric for that: `[^a-zA-z0-9_]`